x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/session/Configuration.java

```diff
@@ -79,8 +79,46 @@ public Configuration(
         int resultTruncationMaxSizeTimeseries,
         int resultTruncationDefaultSizeTimeseries
     ) {
+        this(
+            zi,
+            locale,
+            username,
+            clusterName,
+            pragmas,
+            resultTruncationMaxSizeRegular,
+            resultTruncationDefaultSizeRegular,
+            query,
+            profile,
+            tables,
+            queryStartTimeNanos,
+            allowPartialResults,
+            resultTruncationMaxSizeTimeseries,
+            resultTruncationDefaultSizeTimeseries,
+            Clock.system(zi.normalized())
```

Comment:
```suggestion
            null
```
to initialise just in one place.